plugin/src/openshift/pages/IstioConfigListPage.tsx

```diff
@@ -175,7 +173,13 @@ const IstioConfigListPage = () => {
 
   // Fetch the Istio configs, convert to istio items and map them into flattened list items
   const fetchIstioConfigs = React.useCallback(async (): Promise<IstioConfigItem[]> => {
-    const validate = store.getState().statusState.istioEnvironment.istioAPIEnabled;
+    const validate = await promises
+      .register('getStatus', API.getStatus())
+      .then(response => response.data.istioEnvironment.istioAPIEnabled)
+      .catch(error => {
+        setLoadError({ title: error.response?.statusText, message: error.response?.data.error });
+        return false;
+      });
 
```

Comment:
Why can't we look at the store for this information, and now call back to the server for information that we already have in the client?

Reply:
Ah, right, this is a native page.  And no not wrapped in KialiController or the new KialiContainer.  I get it, thanks.